CondFormats/JetMETObjects/src/SimpleJetCorrector.cc

```diff
@@ -78,17 +79,25 @@ float SimpleJetCorrector::correctionBin(unsigned fBin, const std::vector<float>&
     handleError("SimpleJetCorrector", sserr.str());
   }
   unsigned N = fY.size();
+
   if (N > 4) {
     std::stringstream sserr;
     sserr << "two many variables: " << N << " maximum is 4";
     handleError("SimpleJetCorrector", sserr.str());
   }
   const std::vector<float>& par = mParameters.record(fBin).parameters();
 
-  double params[par.size() - 2 * N];
+  auto nParams = std::max<int>(static_cast<int>(par.size()) - 2 * N, 1);
+  double params[nParams];
   for (unsigned int i = 2 * N; i < par.size(); i++) {
     params[i - 2 * N] = par[i];
   }
+  if (nParams < 1) {
```

Comment:
What is the purpose of this check? Line 90 guarantees `nParams => 1`.

Reply:
@laurenhay , this check is always false

Reply:
Yes I'm removing it

Reply:
Hi @laurenhay did you have the chance to change this? Please squash the new commit when you make it. Thanks